libs/labelbox/src/labelbox/client.py

```diff
@@ -2216,3 +2216,27 @@ def get_embedding_by_name(self, name: str) -> Embedding:
                 return e
         raise labelbox.exceptions.ResourceNotFoundError(Embedding,
                                                         dict(name=name))
+
+    def upsert_label_feedback(self, label_id: str, feedback: str,
+                              scores: Dict[str, float]) -> Entity.Label:
+        """
+
+        Args:
+            label_id: Target label ID
+            feedback: Free text comment regarding the label
+            scores: A dict of scores, the key is a score name and the value is the score value
+        Returns: A list of LabelScore instances
+
+        """
+        mutation_str = """mutation UpsertAutoQaLabelFeedbackPyApi($labelId: ID!, $feedback: String!, $scores: Json!){
```

Comment:
would be nice to keep mutation string formatted to keep style consistency across the file